lib/swm050/Makefile

```diff
@@ -33,7 +33,7 @@ TGT_CFLAGS      += $(DEBUG_FLAGS)
 TGT_CFLAGS	+= $(STANDARD_FLAGS)
 ARFLAGS		= rcs
 
-OBJS += gpio.o
+OBJS += gpio.o clk.o flash.o pwr.o
```

Comment:
this is fine, but to avoid conflicts merging this sort of thing, I've moved most other families to one file/common set per line, in alphabetical order, so
```
OBJS += clk.o
OBJS += flash.o
OBJS += gpio.o
....
```

Reply:
Fixed.